javav2/example_code/appsync/Readme.md

```diff
@@ -0,0 +1,61 @@
+# AWS AppSync code examples for the AWS SDK for Java
```

Comment:
AWS AppSync code examples for the SDK for Java

Reply:
fixed